dataactcore/models/jobTrackerInterface.py

```diff
@@ -275,3 +275,6 @@ def getSubmissionStatus(self,submissionId):
         if statuses["finished"] == len(jobIds)-skip_count: # need to account for the jobs that were skipped above
             return "finished"
         return "unknown"
+
+    def getSubmissionsByUserId(self, user_id):
+        return self.session.query(Submission).filter(Submission.user_id == user_id).all()
```

Comment:
I can move it. We could also leave it and remove it here since the cgac script isn't a permanent solution.